### PR TITLE
Integrate Codeclimate

### PR DESCRIPTION
**Pull request by Reimirno**
Downgrade simplecov to below 0.17.1
Removed cobertura

### CODE REVIEW

**Comment by Reimirno** [comment]
Thanks to Professor Ball's help this is resolved.

We downgraded `simplecov` to 0.17.1 (the last version without compatibility issue).

I fixed a few syntax issues and made some necessary changes in response to the downgrade. 

Now it should be all fine to merge.

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require "simplecov"
-require "simplecov-cobertura"
+# require "simplecov-cobertura"
```
Comment (cycomachead) [inline spec/rails_helper.rb:5]:
```suggestion
```

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-SimpleCov.formatter = SimpleCov::Formatter::CoberturaFormatter
+# SimpleCov.formatter = SimpleCov::Formatter::CoberturaFormatter
```
Comment (cycomachead) [inline spec/rails_helper.rb:7]:
```suggestion
```

File: features/support/env.rb [hunk 3]
```diff
@@ -2,17 +2,17 @@
 
 require "simplecov"
 require "simplecov-console"
-require "simplecov-cobertura"
+# require "simplecov-cobertura"
```
Comment (cycomachead) [inline features/support/env.rb:5]:
```suggestion
```

File: features/support/env.rb [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
-  enable_coverage :branch
+  #   enable_coverage :branch
 
-  formatter SimpleCov::Formatter::MultiFormatter.new([
-    SimpleCov::Formatter::CoberturaFormatter,
-    SimpleCov::Formatter::Console,
-    SimpleCov::Formatter::HTMLFormatter
-  ])
+  #   formatter SimpleCov::Formatter::MultiFormatter.new([
+  #     SimpleCov::Formatter::CoberturaFormatter,
```
Comment (cycomachead) [inline features/support/env.rb:12]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -101,3 +101,11 @@ If bundler install runs successfully, continue with the following commands to co
 - `heroku run rake db:migrate`
 - `figaro heroku:set -e production`
 - `heroku open`
+
+
+### CodeClimate Local Test
+
```
Comment (cycomachead) [inline README.md:107]:
```suggestion
### CodeClimate Local Test
```

File: Gemfile [hunk 6]
```diff
@@ -86,10 +86,10 @@ end
 # setup Cucumber, RSpec, Guard support
 group :test do
   gem "rspec-rails"
-  gem "simplecov", require: false
+  gem "simplecov", "< 0.18", require: false
   gem "simplecov-json", require: false
   gem "simplecov-console", require: false
-  gem "simplecov-cobertura", require: false
+  # gem "simplecov-cobertura", require: false
```
Comment (cycomachead) [inline Gemfile:92]:
```suggestion
```

**Review by cycomachead — CHANGES_REQUESTED** [review]
Looks good, but let's just delete the references to coberturra. 
(you can commit all the suggestions in a batch)

**Comment by Reimirno** [comment]
Committed all the suggestions and squashed.
(I didn't know you could commit suggestions in batch, Ugh. I will do that next time)

**Pull request merged**